node/service/src/lib.rs

```diff
@@ -576,8 +577,14 @@ macro_rules! new_light {
 }
 
 /// Builds a new object suitable for chain operations.
-pub fn new_chain_ops<Runtime, Dispatch, Extrinsic>(mut config: Configuration)
-	-> Result<impl ServiceBuilderCommand<Block=Block>, ServiceError>
+pub fn new_chain_ops<Runtime, Dispatch, Extrinsic>(mut config: Configuration) -> Result<
+	(
+		Arc<service::TFullClient<Block, Runtime, Dispatch>>, 
+		Arc<TFullBackend<Block>>,
+		consensus_common::import_queue::BasicQueue<Block, PrefixedMemoryDB<BlakeTwo256>>,
```

Comment:
Nitpick: Can we use this simplify this type so that we don't need to import `sp_trie` for `PrefixedMemoryDB`?

Reply:
I don't know... maybe someone else would have an idea